specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/AuthConfig"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update Container App AuthConfig": {
+            "$ref": "./examples/AuthConfigs_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ContainerAppsAuthConfigs"
+        ],
+        "summary": "Delete a Container App AuthConfig.",
+        "description": "Description for Delete a Container App AuthConfig.",
+        "operationId": "ContainerAppsAuthConfigs_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Container App AuthConfig.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Container App deleted successfully."
+          },
+          "204": {
+            "description": "Container App AuthConfig does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Container App AuthConfig": {
+            "$ref": "./examples/AuthConfigs_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AuthConfig": {
+      "description": "Configuration settings for the Azure ContainerApp Authentication / Authorization V2 feature.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "SiteAuthSettingsV2 resource specific properties",
```

Comment:
>SiteAuthSettingsV2 [](http://example.com/codeflow?start=26&length=18)

Does it make sense to have V2 in the description?

Reply:
Good suggestion, fixed in update